### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by gandradep**
- Install Ruby on Rails framework.
- Use Ruby Gems as a software packages system.
- Understand what ORM is.
- Use database migration files to maintain database schema.

### CODE REVIEW

File: db/migrate/20230125165622_add_user_ref_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (rloterh) [inline db/migrate/20230125165622_add_user_ref_to_posts.rb]: - [ ] Taking into consideration the given ERD, we can see that `user` model is referenced using foreign key in models as `AuthorId`. Let's, therefore, bear in mind that `author` is aliasing the `user` model, and hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :posts, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :posts do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

File: db/migrate/20230125171843_add_user_ref_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (rloterh) [inline db/migrate/20230125171843_add_user_ref_to_comments.rb]: - [ ] Similarly, the `user` model is referenced using foreign key in models as `AuthorId`, and `author` being an alias of `user` hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :comments, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :comments do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

File: db/migrate/20230125172355_add_user_ref_to_likes.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (rloterh) [inline db/migrate/20230125172355_add_user_ref_to_likes.rb]: - [x] Again, the `user` model is referenced using foreign key in models as `AuthorId`, and `author` being an alias of `user` hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :likes, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :likes do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

File: README.md [hunk 4]
```diff
@@ -99,135 +50,50 @@ After you're finished please remove all the comments and instructions!
...
-
-```
- -->
+To run the project, just create the database and the schema file should create the tables.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
```
Comment (rloterh) [inline README.md:80]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Gabriel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   🏃 

## Highlights 👍  
- Nice job so far implementing migration query

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230125172355_add_user_ref_to_likes.rb (hunk 3 above)
Comment (gandradep) [reply db/migrate/20230125172355_add_user_ref_to_likes.rb]: @rloterh Thanks for the review. Already fixed the column name of that foreign key.

**Review by mikethreels — APPROVED** [review]
Hi Gabriel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gandradep** [comment]
@mikethreels Thanks for the review.

**Pull request merged**